src/canvas-extractor/edfi_canvas_extractor/mapping/authentication_events.py

```diff
@@ -76,10 +78,9 @@ def map_to_udm_system_activities(authentication_events: pd.DataFrame) -> pd.Data
     df["ActivityStatus"] = "active"
     df["ParentSourceSystemIdentifier"] = ""
     df["ActivityTimeInMinutes"] = ""
-    df["SourceCreateDate"] = df["ActivityDateTime"]
     df["SourceLastModifiedDate"] = ""
-    df["ActivityDateTime"] = pd.to_datetime(df["ActivityDateTime"]).dt.strftime(
-        constants.DATE_FORMAT
-    )
```

Comment:
Previously, "SourceCreateDate" copied from "ActivityDateTime" before the conversion. Now the conversion occurs first, using a shared function.